pyseq.py

```diff
@@ -66,9 +66,12 @@
 # regex for matching format directives
 format_re = re.compile(r'%(?P<pad>\d+)?(?P<var>\w+)')
 
+# character to join explicit frame ranges on
+range_join = os.environ.get("PYSEQ_SEP", " ")
```

Comment:
Use range_join as an env var so people can have there own separator for broken image sequences.